avocado/utils/astring.py

```diff
@@ -35,6 +36,8 @@
 from six.moves import xrange as range
 
 
+#: Preferred encoding (could be used as default)
+ENCODING = locale.getpreferredencoding()
```

Comment:
This is a bad idea because it will document the result value in our docs as a "constant", but in reality it will be different based on the the user's environment.  Example:

![image](https://user-images.githubusercontent.com/447178/39473531-499a626a-4d1d-11e8-8f1c-ee84cb20899b.png)

Reply:
I'm not aware of any way to trick sphinx to a different value, but we can always document in text that it's evaluated based on the sytem locales.

For example the `avocado.utils.runtime.CURRENT_JOB` reports `None`, etc. I don't see this as a big issue. On the other hand while working on yaml2mux I found several modules where I had to add the `locale.getpreferredencoding`, usually a module-wise as several places where it's needed were present so having it as a single import (especially when `astring` was usually also imported) would be useful.

Also don't forget that people are creative so having a default preferred encoding in utils should unify things project-wise.